plugins/services/src/js/components/ServiceBreadcrumbs.js

```diff
@@ -236,7 +236,7 @@ class ServiceBreadcrumbs extends React.Component {
 
 ServiceBreadcrumbs.propTypes = {
   extra: React.PropTypes.arrayOf(React.PropTypes.node),
-  serviceID: React.PropTypes.string.isRequired,
+  serviceID: React.PropTypes.string,
```

Comment:
Since this is not required shouldn't we provide some reasonable defaults?
Right now if it's missing, the `trimmedServiceID` would be `undefined`, right?